### PR TITLE
Add active_record adapter and adapter tests

### PR DESCRIPTION
**Pull request by edpaget**
this also makes it compatible with the latest panoptes schema.

### CODE REVIEW

File: lib/cellect/server/adapters/panoptes.rb [hunk 1]
```diff
@@ -1,76 +1,71 @@
...
+        class UserSeenSubject < ActiveRecord::Base
+        end
+      end
+
       class Panoptes < Default
         def initialize
-          @pg ||= ConnectionPool.new(size: ENV.fetch('PG_POOL_SIZE', 25).to_i) do
-            PG.connect connection_options
-          end
+          ActiveRecord::Base.establish_connection(**connection_options)
```
Comment (camallen) [inline lib/cellect/server/adapters/panoptes.rb]: I think my only concern with the PR is using connection pooling for threaded puma. Any thoughts?

Comment (edpaget) [inline lib/cellect/server/adapters/panoptes.rb]: I'm pretty sure AR creates a pool with 5 connections by default. We can configure more connections with the `pool:` key.

File: lib/cellect/server/adapters/panoptes.rb [hunk 2]
```diff
@@ -1,76 +1,71 @@
...
-        def with_pg
-          @pg.with{ |pg| yield pg }
-        end
-
         def connection_options
           {
+            adapter: "postgresql",
             host: ENV.fetch('PG_HOST', '127.0.0.1'),
             port: ENV.fetch('PG_PORT', '5432'),
             dbname: ENV.fetch('PG_DB', 'cellect'),
```
Comment (camallen) [inline lib/cellect/server/adapters/panoptes.rb]: Can you add a pool option here that inherits from the global config? https://github.com/zooniverse/cellect_panoptes/blob/9140f6933aac7df17d730ffe2d00f177f57a028a/start#L27

Comment (edpaget) [inline lib/cellect/server/adapters/panoptes.rb]: done

**Pull request merged**